src/connectedComponents/ToolbarRow.js

```diff
@@ -203,13 +229,17 @@ function _getButtonComponents(toolbarButtons, activeButtons) {
 function _handleToolbarButtonClick(button, evt, props) {
   if (button.commandName) {
     const options = Object.assign({ evt }, button.commandOptions);
+    _setActiveToolAsPassive.call(this);
```

Comment:
This shouldn't be called if the button clicked is an action rather than a tool activation.

Reply:
Indeed, thanks for catching it! Updated.